### ISSUE TITLE
 [private.vpn.jenkins.io] `2024-09-06` (September 2024) VPN CRL expires

### ISSUE THREAD

**Issue by dduportal** [jenkins-infra/helpdesk#4266]
### Service(s)

VPN

### Summary

The Jenkins Infra calendar reminds us that the VPN CRL expires the 6 September 2024.

Todo list for the team member who will take this action:

- [x] Read https://github.com/jenkins-infra/docker-openvpn#howto-review-certificate-revocation-list and check that you have all the pre-requisites
- [x] Open a PR to jenkins-infra/docker-openvpn that updates the CRL - https://github.com/jenkins-infra/docker-openvpn/pull/351
- [x] Once the PR is merged, ensures that there is a new version of the container automatically built and released
- [x] Run manually the updatecli build for jenkins-infra/jenkins-infra to have an automated PR to bump the VPN image version
- [x] Once deployed, connect to the `private.vpn.jenkins.io` VM with SSH (requires your public Ip allowed to do that) and ensure that the new CRL is present and updated
- [x] Create a new team calendar event to the date of the next expiration, with alerts 3 weeks before and 2 weeks before

Previous issues: https://github.com/jenkins-infra/helpdesk/issues/3983

Nice to have: is there a way to have automated PR with `updatecli` in the docker-openvpn repository?
